test/integration/cdi/example-driver/app/kubeletplugin.go

```diff
@@ -114,8 +110,10 @@ func (ex *examplePlugin) NodePrepareResource(ctx context.Context, req *drapbv1.N
 	}
 
 	spec := specs.Spec{
-		Version: cdiVersion,
+		Version: "0.2.0",
```

Comment:
@klihub: what is the "right" version string to use here?

Isn't it defined indirectly through the Go API, i.e. `specs.Spec`, `specs.Device`, etc.?

Reply:
Moved to https://github.com/container-orchestrated-devices/container-device-interface/issues/60#issuecomment-1212889973 and added a TODO to the code.